composer.json

```diff
@@ -1,8 +1,7 @@
 {
-    "name": "prestashop/composer-module-plugin",
+    "name": "prestashop/composer-script-handler",
```

Comment:
it's weird that the package name doesn't match the repo no?

Reply:
I know, I'll ask @eternoendless to update it (it was supposed to be a plugin, now a script handler)